node/mdSchema.ts

```diff
@@ -37,6 +37,14 @@ export const COST_CENTER_FIELDS = [
 ]
 export const COST_CENTER_SCHEMA_VERSION = 'v0.0.6'
 
+export const B2B_SETTINGS_DATA_ENTITY = 'b2b_settings'
```

Comment:
Instead of storing settings in Masterdata, VBASE would be a better solution and more consistent with our other B2B apps. See this example in B2B Quotes: https://github.com/vtex-apps/b2b-quotes-graphql/blob/master/node/resolvers/mutations/index.ts#L517